math/src/fft/tests.rs

```diff
@@ -22,8 +23,8 @@ fn fft_in_place() {
     let domain = build_domain(n);
     let expected = polynom::eval_many(&p, &domain);
     let twiddles = super::get_twiddles::<BaseElement>(n);
-    super::serial::fft_in_place(&mut p, &twiddles, 1, 1, 0);
-    super::permute(&mut p);
+    p.as_mut_slice().fft_in_place(&twiddles);
+    p.as_mut_slice().permute();
```

Comment:
Do we actually need to do `as_mut_slice()` here? Seems like it works fine on line 67 below without it.

Reply:
yes- we don't need to do `as_mut_slice()`. I missed this here. Thanks.